### ISSUE TITLE
Gatsby V3

### ISSUE THREAD

**Issue by muuvmuuv** [Js-Brecht/gatsby-plugin-pnpm#6]
Gatsby released version 3 recently. It now prints this warning. I could not test if this plugin still works since I am doing a huge rewrite of my project but maybe someone else can.

```
warn Plugin gatsby-plugin-pnpm is not compatible with your gatsby version 3.0.0 - It requires gatsby@~2.x.x
```

**Comment by Js-Brecht** [comment]
It's just a peer dependency warning.  I can't imagine the upgrade will affect this plugin, unless they've changed the webpack config api.  Webpack 5 accepts aliases/module resolvers the same way Webpack 4 does, so I think this should be good.

Once I have an opportunity to upgrade some projects to Gatsby v3 and make sure everything works, I can come back and update the peer dependency.  Until then, if somebody can verify it is working or if anything breaks, please let me know

**Comment by gchallen** [comment]
I attempted the upgrade a few days ago. I found that `react-fast-refresh` was crashing afterward and pages wouldn't hot reload. I have no idea if this is pnpm problem or not, but I didn't see it reported in the main Gatsby issues page, and this is one of the more unusual parts of my configuration, so I thought I would note it.

**Comment by Js-Brecht** [comment]
@gchallen Can you share a reproduction?

**Comment by gchallen** [comment]
Yeah, sorry, I suck, no. At least not quickly. My project is a big ugly poorly-organized monorepo.

I'll keep trying to upgrade once I see some additional changes to the Gatsby sources. Feel free to ignore at this point.

**Comment by gchallen** [comment]
Actually this seem like it's been reported on the main Gatsby issues page now with 3.0.0+. So very likely not related to pnpm. I should have figured 😄. Sorry for the spam.

**Comment by muuvmuuv** [comment]
As far as I can see, it does work without problems.

**Comment by Js-Brecht** [comment]
Published in v1.2.6
